data-scope-main/src/main/resources/static/webview-ui/src/mocks/datasource.ts

```
@@ -0,0 +1,907 @@
+        switch (column.type) {
+          case 'INT':
+            row[column.name] = Math.floor(Math.random() * 10000)
+            break
+          case 'VARCHAR':
+            row[column.name] = `Value-${Math.random().toString(36).substring(2, 7)}`
+            break
+          case 'DECIMAL':
+            row[column.name] = +(Math.random() * 1000).toFixed(2)
+            break
+          case 'TIMESTAMP':
+            row[column.name] = new Date(Date.now() - Math.floor(Math.random() * 30) * 24 * 60 * 60 * 1000).toISOString()
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)